packages/webpack/src/buildtime/emitSes.js

```diff
@@ -5,30 +5,28 @@ const {
 } = require('webpack')
 
 const lockdownSource = readFileSync(
-  path.join(require.resolve('ses'), '../lockdown.umd.min.js'),
```

Comment:
Need to remove now unused `path` import to satisfy linting

https://github.com/LavaMoat/LavaMoat/actions/runs/10849844996/job/30109919317?pr=1296